.bcr/presubmit.yml

```diff
@@ -1,5 +1,5 @@
 bcr_test_module:
-  module_path: "e2e/smoke"
+  module_path: "e2e/crane_as_registry"
```

Comment:
I don't think we should change this, we use "smoke" in all our rulesets as the e2e that BCR presubmit runs. Can we just green up that one on windows?

Reply:
I could switch places. make smoke test use crane and the new test cover zot

Reply:
on second thought, that would make us recommend crane as the default registry, not sure if that's what we want. i don't want users memory to be filled with big blobs

Reply:
done